IDE/MDK5-ARM/Projects/CryptBenchmark/CryptBenchmark.uvguix

```diff
@@ -0,0 +1,3403 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
I'm guessing all *.uvguix files can be excluded? It looks like a user-specific GUI file, which is quite large.